e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/add-to-an-existing-payment.spec.js

```diff
@@ -47,40 +39,23 @@ context(`${PDC_TEAMS.PDC_RECONCILE} users can add fee records to existing paymen
     .withPaymentExchangeRate(2)
     .withStatus('TO_DO')
     .build();
-  const feeRecordTwo = FeeRecordEntityMockBuilder.forReport(report)
+  const secondFeeRecord = FeeRecordEntityMockBuilder.forReport(report)
     .withId(FEE_RECORD_ID_TWO)
     .withFacilityId('22222222')
     .withExporter('Exporter 2')
     .withPaymentCurrency(PAYMENT_CURRENCY)
     .withFeesPaidToUkefForThePeriod(200)
     .withFeesPaidToUkefForThePeriodCurrency('EUR')
     .withPaymentExchangeRate(0.5)
-    .withPayments([paymentOne])
-    .withStatus('DOES_NOT_MATCH')
-    .build();
-  const feeRecordThree = FeeRecordEntityMockBuilder.forReport(report)
-    .withId(FEE_RECORD_ID_THREE)
-    .withFacilityId('33333333')
-    .withExporter('Exporter 3')
-    .withPaymentCurrency(PAYMENT_CURRENCY)
-    .withFeesPaidToUkefForThePeriod(300)
-    .withFeesPaidToUkefForThePeriodCurrency('EUR')
-    .withPaymentExchangeRate(0.5)
-    .withPayments([paymentTwo])
+    .withPayments([firstPayment])
     .withStatus('DOES_NOT_MATCH')
     .build();
 
   before(() => {
     cy.task(NODE_TASKS.REINSERT_ZERO_THRESHOLD_PAYMENT_MATCHING_TOLERANCES);
   });
 
-  beforeEach(() => {
-    cy.task(NODE_TASKS.REMOVE_ALL_UTILISATION_REPORTS_FROM_DB);
-    cy.task(NODE_TASKS.REMOVE_ALL_PAYMENTS_FROM_DB);
-    cy.task(NODE_TASKS.INSERT_UTILISATION_REPORTS_INTO_DB, [report]);
-
-    cy.task(NODE_TASKS.INSERT_FEE_RECORDS_INTO_DB, [feeRecordOne, feeRecordTwo, feeRecordThree]);
-
+  const navigateToAddToExistingPaymentScreenGivenSelectedFeeRecord = () => {
```

Comment:
I'm not sure I follow the name of this since we select the fee record inside this helper don't we? Should it be: `navigateToAddToExistingPaymentScreenForFirstFeeRecord`?